app/views/general_shopping_list/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1>Shopping List for <%= @recipe.name %></h1>
+<div class="d-flex flex-row justify-content-between">
+  <p style="font-size: 1.6rem;">Amount of food items to buy: <%= @counter %></p>
+  <p style="font-size: 1.6rem;">
+    Total value of food needed: $<%= @food_price %>
+  </p>
+</div>
+
+<div class="table-responsive">
+  <table class="table table-striped">
+    <thead>
+      <tr>
+        <th>Food</th>
+        <th>Quantity</th>
+        <th>Price</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <% @shopping_list_array.each do |shopping_item| %>
+        <tr>
+          <th><%= shopping_item[0] %></th>
+          <th><%= shopping_item[1] %></th>
```

Comment:
[OPTIONAL] Would it make sense to add the measurement units of the quantity as well?